docs/en/query_language/alter.md

```diff
@@ -165,16 +169,14 @@ So you can put data to the 'detached' directory on one replica, and use the `ALT
 #### REPLACE PARTITION {#alter_replace-partition}
 
 ``` sql
-ALTER TABLE table2_name REPLACE PARTITION partition_expr FROM table1_name
+ALTER TABLE table2 REPLACE PARTITION partition_expr FROM table1
 ```
 
-This query copies the data partition from the `table1` to `table2`. Note that:
+This query copies the data partition from the `table1` to `table2` (data isn't deleted from `table1`). Note that:
```

Comment:
This comment in brackets would look better in the list after "Not that:"

Reply:
I'm not sure, because in a note section we list the main conditions required for the query performing.
The information about data is just a query behavior description. Let's say about it in a separate sentence?

Reply:
It sounds like "Note that" should be replaced with more specific introduction to that list.